app/views/posts/show.html.erb

```diff
@@ -5,6 +5,8 @@
   <body>
   <%= render 'single_post' %>
     <%= render_comments_section(@post) %>
+    <%= render 'shared/like' %>
```

Comment:
- [ ] when i click on the like button, like count does not increment, kindly fix this

Reply:
Hello @VICTOR-CHUKWU 

Thank you for your feedback. I have made the correction. Now, it works as per requirement.

Regards

Reply:
This change has been done well